Documentation/contributing.rst

```diff
@@ -812,303 +960,370 @@ Have a question about how the tests work or want to chat more about improving th
 testing infrastructure for Cilium? Hop on over to the
 `testing <https://cilium.slack.com/messages/C7PE7V806>`_ channel on Slack.
 
-Building Documentation
-----------------------
+.. _howto_contribute:
 
-The documentation has several dependencies which can be installed using pip:
+How to contribute
+-----------------
 
-::
+Getting Started
+~~~~~~~~~~~~~~~
 
-    $ pip install -r Documentation/requirements.txt
+#. Make sure you have a `GitHub account <https://github.com/signup/free>`_
+#. Clone the cilium repository
 
-Whenever making changes to Cilium documentation you should check that you did not introduce any new warnings or errors, and also check that your changes look as you intended.  To do this you can build the docs:
+   ::
+  
+      go get -d github.com/cilium/cilium
+      cd $GOPATH/src/github.com/cilium/cilium
 
-::
+#. Set up your :ref:`dev_env`
+#. Check the GitHub issues for `good tasks to get started
+   <https://github.com/cilium/cilium/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue>`_.
 
-    $ make -C Documentation html
+Preparing Changes for Submission
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After this you can browse the updated docs as HTML starting at
-``Documentation\_build\html\index.html``.
+#. Fork the Cilium repository to your own personal GitHub space or request
+   access to a Cilium developer account on Slack
+#. Push your changes to the topic branch in your fork of the repository.
+#. Submit a pull request on https://github.com/cilium/cilium
+  
+   #. Provide a high level description of the PR, feel free to refer to
+      individual commit messages for details. Remember that the commit
+      descriptions will be used for the git revision history, not the PR
+      description text.
+
```

Comment:
Add a paragraph here WRT labels. (wait, the external contributor can't set labels so never mind)